.vscode/settings.json

```diff
@@ -9,7 +9,7 @@
     },
     "eslint.enable": false,
     "flow.enabled": false,
-
+    "editor.formatOnSave": true,
```

Comment:
Nope, I'm a compulsive saver too, all these things are done async too

There is an option to run on every newline instead, but I felt that was a bit overkill